utils/evaluate/evaluate.go

```diff
@@ -168,3 +168,20 @@ func asFloat(in interface{}) float64 {
 	}
 	panic(fmt.Sprintf("asFloat() not supported on %v %v", reflect.TypeOf(in), in))
 }
+
+// Check whether two slices of type string are equal or not.
+func Equal(a, b []string) bool {
```

Comment:
This method seems not used anywhere in the PR. Shall we remove it?

Reply:
It's actually getting used [here](https://github.com/argoproj/argo-rollouts/pull/1380/files#diff-bc2f3a4d5d2fd799a169ceccb8bf97ae8b8a45f6a69009525c24e41fd846ef5eR644) while comparing the SNI Hosts list from the Rollout Def and the actual Istio VirtualService.

Reply:
The link doesn't point to the right place and search in the browser does not show me anything.

Reply:
Sorry about that. It's **L#644** in file `rollout/trafficrouting/istio/istio.go`.

<img width="718" alt="Screen Shot 2021-08-10 at 4 28 29 PM" src="https://user-images.githubusercontent.com/12120828/128930527-b88e2742-052a-458d-800b-a8b8b00e4de8.png">